.github/workflows/ci.yml

```diff
@@ -34,3 +34,18 @@ jobs:
 
       - name: Check formatting
         run: crystal tool format --check
+
+  testfreebsd:
+    runs-on: macos-latest
+    name: Test (freebsd)
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Crystal and test
+      id: test
+      uses: vmactions/freebsd-vm@v0.1.3
+      with:
+        usesh: true
+        prepare: pkg install -y libyaml libevent crystal shards gmake
+        run: |
+          shards install
+          gmake test
```

Comment:
That would be better fitting under the `matrix.include` key.

Reply:
Not sure why that is needed, since the matrix allows to spawn several jobs which is not planned here.